resource_customizations/argoproj.io/Rollout/actions/action_test.yaml

```diff
@@ -35,6 +39,8 @@ discoveryTests:
       disabled: true
     - name: promote-full
       disabled: true
+    - name: pause
+      disabled: false
```

Comment:
The name of the inputPath makes me think this might have to be true, since it's already paused. But not sure what's nil means there.

Reply:
True. Fixed it now.

Reply:
`nil_paused` implies no "paused" spec is present in the rollout spec and hence the rollout is not paused. 
Therefore, it should enabled.